app/src/androidTest/java/com/mapbox/maps/testapp/overlay/MapOverlayPluginTest.kt

```diff
@@ -29,7 +29,7 @@ class MapOverlayPluginTest : BaseMapTest() {
   @Before
   fun setUp() {
     super.before()
-    mapOverlayPlugin = mapView.overlay()
+    mapOverlayPlugin = mapView.mapboxOverlay
```

Comment:
why not just `overlay`?

Reply:
Have described in pr description, it would conflict with ViewGroup.getOverlay()